packages/nimble-components/src/chip/specs/README.md

```diff
@@ -0,0 +1,200 @@
+# Nimble Chip
+
+## Overview
+
+The `nimble-chip` is a component that presents text, an optional "remove" button, optional slotted content to the left of the text, and an optional error icon.
```

Comment:
I think this overview section shouldn't mention the optional error icon since that's being considered out of scope for the initial creation of the component.